ee/codegen/src/project.ts

```diff
@@ -186,14 +192,22 @@ ${errors.slice(0, 3).map((err) => {
       // inputs.py
       inputs.persist(),
       // workflow.py
-      workflow.getWorkflowFile().persist(),
+      workflowFile.persist(),
       // nodes/*
       ...this.generateNodeFiles(nodes),
     ]);
 
     // error.log
     await this.generateErrorLogFile().persist();
 
+    // sandbox.py
+    if (this.sandboxInputs) {
+      await this.generateSandboxFile({
+        workflowFile,
+        inputsFile: inputs,
+      }).persist();
+    }
```

Comment:
[blocking] We should be generating both of these in parallel in the `Promise.all` above to minimize latency of codegen.

Reply:
Error log I think needs to be after bc it collects the errors of all the above